modules/containers-signature-verify-unsigned.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-container-signature.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="containers-signature-verify-artifacts_{context}"]
+= Understanding the verification of container images lacking verifiable signatures
+Each {product-title} release image is immutable and signed with a Red Hat production key. During an {product-title} update or install, a release image might deploy container images that do not have verifiable signatures. Each signed release image digest is immutable. Each reference in the release image is to the immutable digest of another image, so the contents can be trusted transitively. In other words, the signature on the release image validates all release contents.
```

Comment:
Per the [ISG word usage](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#i) entry on "install": "Do not use as an adjective or a noun. Use installation."

```suggestion
Each {product-title} release image is immutable and signed with a Red Hat production key. During an {product-title} update or installation, a release image might deploy container images that do not have verifiable signatures. Each signed release image digest is immutable. Each reference in the release image is to the immutable digest of another image, so the contents can be trusted transitively. In other words, the signature on the release image validates all release contents.
```